src/reanimated2/Hooks.js

```diff
@@ -268,6 +268,7 @@ export function useAnimatedStyle(updater, dependencies) {
   const initRef = useRef(null);
   const inputs = Object.values(updater._closure);
   const viewRef = useRef(null);
+  const viewName = useSharedValue(null);
```

Comment:
same comment as before. Lets keep viewTag and viewName at least next to each other. I'd prefer if we just have a descriptor that stores both, this way we'd also avoid additional hook call here.

Reply:
fixed b3f674c646c6d7972e8a0ed50226c8028433ac16